internal/db/migrations/005_reservations.sql

```diff
@@ -11,6 +11,12 @@ CREATE TABLE reservations
   UNIQUE (id, provider)
 );
 
+CREATE TABLE instance_reservation
+(
+  reservation_id BIGINT NOT NULL REFERENCES reservations(id) ON DELETE CASCADE,
+  instance_id TEXT NOT NULL UNIQUE
+);
+
```

Comment:
We have missed a small detail: all our tables are plural, this is singular. Fixed in: https://github.com/RHEnVision/provisioning-backend/pull/144